packages/snack-sdk/src/transports/TransportImplSocketIO.ts

```diff
@@ -42,7 +42,7 @@ export default class TransportImplSocketIO extends TransportImplBase {
     this.socket = io(this.snackpubURL, { transports: ['websocket'] });
 
     this.socket.on('connect', () => {
-      this.socket?.emit('subscribeChannel', { channel: this.channel, sender: this.socket?.id });
+      this.socket?.emit('subscribeChannel', { channel: this.channel, sender: this.socket?.id! });
```

Comment:
Happy to change these to type guards if that is the preference in this situation.